lib/Raven/Serializer.php

```diff
@@ -67,17 +67,44 @@ public function __construct($mb_detect_order = null)
      * sanitization and encoding.
      *
      * @param mixed $value
+     * @param array $context
+     * @return string|bool|double|int|null|object|array
```

Comment:
Accept — of course. But we are talking about param and return types, aren\'t we? Of course we should change param types to "mixed". I will do it. But what about return types?